.github/workflows/compatibility-test.yml

```diff
@@ -5,7 +5,7 @@ on: pull_request
 jobs:
   compatibility_test:
     # https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
Does the latest ubuntu version contain jdk 1.8?

Reply:
Good Question!

Just checked that it does have it.
https://github.com/actions/runner-images#available-images `ubuntu-latest` points to 22.04,

and 22.04 runner image has Java 8 -
https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#java